src/braket/experimental/autoqasm/program/program.py

```diff
@@ -84,6 +89,22 @@ def __init__(self, oqpy_program: oqpy.Program, has_pulse_control: bool = False):
         self._oqpy_program = oqpy_program
         self._has_pulse_control = has_pulse_control
 
+    def bind_calibrations(self, gate_calibrations: Union[Callable, List[Callable]]) -> Program:
+        """Binds the gate calibrations to the program.
+
+        Args:
+            gate_calibrations (Union[Callable, List[Callable]]): The gate calibrations to bind.
+        """
```

Comment:
This docstring seems like it should include a `Returns`? I wonder if the bcs linting is not working for some reason - do you get any error when running tox linters or `flake8 --enable-extensions=BCS -v`?

Reply:
It didn't works until I reset my tox environment. Works now.